argo-cd-apps/overlays/environment/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base/all-clusters
```

Comment:
don't you need to include the new `environment` kustomization dir?

Reply:
changed approach a little bit, please check now